shavar/templates/swagger.mako

```diff
@@ -0,0 +1,100 @@
+swagger: "2.0"
+host: ${HOST}
+info:
+  title: Shavar Service
+  description: Mozilla's implementation of the safe browsing protocol
```

Comment:
It's not just used by tracking protection anymore, so it makes sense to just refer to the protocol.